### PR TITLE
Adds an invisible skip link at the head of pages

### PR DESCRIPTION
**Pull request by kharbutli-meta**
## Description

GTT-1695: Add an invisible skip link at the beginning of every page that allows users to skip the header content and jump to the main content. The link appears when focused on. This is to support users with screen readers and/or those without a mouse.

Screen Shot 2022-02-10 at 2 18 34 PM

## Testing

* Unit tests.
* Local tests.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: frontend/src/layouts/Admin.tsx [hunk 1]
```diff
@@ -64,6 +64,11 @@ function AdminLayout(props: LayoutProps) {
       <Header className="usa-header usa-header--basic">
         <div className="usa-nav-container">
           <div className="usa-navbar navbar-long">
+            <div>
+              <a className="skip-link" href="#main">
```
Comment (mabreuortega) [inline frontend/src/layouts/Admin.tsx]: USWDS supports skipnavs by default

```
<a class="usa-skipnav" href="#main-content">Skip to main content</a>
```

File: frontend/src/layouts/Public.tsx [hunk 2]
```diff
@@ -42,6 +43,11 @@ function PublicLayout(props: LayoutProps) {
       <Header className="usa-header usa-header--basic">
         <div className="usa-nav-container">
           <div className="usa-navbar navbar-long">
+            <div>
+              <a className="skip-link" href="#main">
+                {t("SkipToMainContent")}
```
Comment (mabreuortega) [inline frontend/src/layouts/Public.tsx:47]: ditto

**Review by mabreuortega — COMMENTED** [review]
Suggestion made

File: frontend/src/layouts/Public.tsx (hunk 2 above)
Comment (kharbutli-meta) [reply frontend/src/layouts/Public.tsx:47]: Done.

File: frontend/src/layouts/Admin.tsx (hunk 1 above)
Comment (kharbutli-meta) [reply frontend/src/layouts/Admin.tsx]: Done.

**Review by mabreuortega — APPROVED** [review]
LGTM

**Pull request merged**